tests/config_notebooks_tests/annotation.config.json

```diff
@@ -1,7 +1 @@
-{
-    "save_path": "tests/data/",
-    "n_epochs": 1,
-    "n_labelled_samples_per_class": 3,
-    "M_permutation": 10,
-    "M_sampling": 1
-}
+{"save_path": "/tmp/pytest-of-jules/pytest-0/temp_data/", "n_epochs": 1, "n_labelled_samples_per_class": 3, "M_permutation": 10, "M_sampling": 1}
```

Comment:
Can we avoid hard coding this path somehow? We'd use the temporary directory pytest creates ideally.

Reply:
I'm not sure we can always write a directory called `/tmp/pytest-of-jules`. I don't think windows systems even have a `/tmp` directory.

Reply:
On the other hand, I'm not quite sure what the right way to do it is, and I think this PR overall is a nice improvement, so let's get it merged.